config/defaults.conf

```diff
@@ -87,8 +87,8 @@ reload_indexing = 0  // If true, force the rebuilding of the index files in prep
                      // exist.
 target_train_data_fraction = 1  // Use only the specified fraction of the training data in the do_target_task_training phase.
                         // should not impact pretraining, even for the same task.
-                        // pretrain_data_fraction should not be used together with                 // target_train_data_fraction
-                        // both should not be less then 1 at a same time
+                        // pretrain_data_fraction can be used together with target_train_data_fraction. Both the fractions can be less then 1 at a same time.
```

Comment:
The second half of this line is confusing—just delete it. Also, try to keep lines under 100 chars.